tools/android/BUILD.tools

```diff
@@ -108,8 +108,8 @@ genrule(
     name = "ijar_desugared_java8_legacy_libs",
     srcs = ["desugared_java8_legacy_libs"],
     outs = ["desugared_java8_legacy_libs-ijar.jar"],
-    cmd = "$(location //third_party/ijar) $< $@",
-    tools = ["//third_party/ijar"],
+    cmd = "$(location @remote_java_tools//:ijar_binary) $< $@",
+    tools = ["@remote_java_tools//:ijar_binary"],
```

Comment:
This is precompiled, right? How would multiplaform builds work?

Reply:
I've taken care of that with platform specific remote java tools. Now this will retrieve the platform specific precompiled binary.